modules/must-gather-storage-limit.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="must-gather-storage-limit_{context}"]
+= Changing the must-gather storage limit
+
+When using the `oc adm must-gather` command to collect data the default maximum storage for the information is 30% of the storage capacity of the container. Once the 30% limit is reached the container is killed and the gathering process stops. Information already gathered will be downloaded to your local storage. You will need either a container with more storage capacity or to adjust the maximum volume percentage before running the must-gather command again.
+
+If you reach the storage limit you will see a message output informing you what happened.
+
+For example:
+
+[source,terminal]
+----
+Disk usage exceeds the volume percentage of %d for mounted directory. Exiting...
+----
+
+.Prerequisites
+
+ifndef::openshift-rosa,openshift-dedicated[]
+* You have access to the cluster as a user with the `cluster-admin` role.
+endif::openshift-rosa,openshift-dedicated[]
+ifdef::openshift-rosa,openshift-dedicated[]
+* You have access to the cluster as a user with the `dedicated-admin` role.
+endif::openshift-rosa,openshift-dedicated[]
+* The {product-title} CLI (`oc`) is installed.
+
+[IMPORTANT]
+====
+If you installed an earlier version of the `oc` binary, you cannot use it to run this command. You must download and install the 4.15 or later version of the `oc` binary.
+====
+
+.Procedure
+
+. Run the `oc adm must-gather` command with the`volume-percentage` flag. The new value cannot exceed 100.
+
+For example:
```

Comment:
I don't think this line is necessary. I'd remove it and put a `+` on the line here so that the code block comes right after the above intro